server/database.py

```diff
@@ -79,6 +79,16 @@ def create_subscriber(db, tag_id, subscriber_id):
     return ({"Success": "Resource Created"}, 201)
 
 
+# (IP) remove_subscriber removes a subscriber from a selected tag id in a database
```

Comment:
What's (IP)? Can we use the same docstring format we're using with other methods?

Reply:
IP just means in progress, I test in discord this until I run it alongside anthony's notification system. In progress is inherent here, so I'll remove it.